github-sync/github-data/sigstore/repositories.yaml

```diff
@@ -153,6 +153,9 @@ repositories:
       - name: Core Team
         id: 4563391
         permission: admin
+      - name: community-codeowners
+        id: 4563391 # TODO: Update once PR is merged
+        permission: maintain
```

Comment:
Given the loss-of-access issues `model-transparency` had with this approach, I would avoid copying the id from `Core Team`

Reply:
Done, thanks